specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/network.json

```diff
@@ -143,6 +143,21 @@
       "description": "Reference to another subresource.",
       "x-ms-azure-resource": true
     },
+    "ChildResource": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Resource ID."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type."
+        }
+      },
+      "description": "Reference to another childresource.",
```

Comment:
`childresource` -> `child resource`

Reply:
Removed this type completely and added type property in specific resource.